### ISSUE TITLE
Can't read in phylip alignments?

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#118]
Crashing when using a phylip alignment as input:
```
./pxlssq -s AA_5tax.phy 
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
Aborted (core dumped)
```
I've checked out an old version, and can confirm it is nothing that I have been doing recently. Should be a simple fix.

**Comment by josephwb** [comment]
Hrm. It seems that _reasonable_ phylip files are read correctly. The error above occurs if not all characters are present on a single line:
```
5 20
TaxonA  AAATTTCCCTGTCCC
        TTTAA
TaxonB  GCTCGAGGGGCCCCA
        AGACC
TaxonC  ACGCTCCCCCTTAAA
        AATGA
TaxonD  TCCTTGTTCAACTCC
        GGTGG
TaxonE  TTACTATTCCCCCCC
        GCCGG
```
It turns out that PAUP does this sort of writing by default (100 chars per line) so this is not necessarily an edge case.

**Comment by josephwb** [comment]
The original format limited sequence names to 10 characters, and the sequence started on the 11th _no matter what_ i.e., there is a good chance there is no whitespace between the name and the sequence. I don't think we should bother trying to support this.

**Comment by josephwb** [comment]
Ugh. This is valid:
```
    6   39
Archaeopt CGATGCTTAC CGCCGATGCT
HesperorniCGTTACTCGT TGTCGTTACT
BaluchitheTAATGTTAAT TGTTAATGTT
B. virginiTAATGTTCGT TGTTAATGTT
BrontosaurCAAAACCCAT CATCAAAACC
B.subtilisGGCAGCCAAT CACGGCAGCC

TACCGCCGAT GCTTACCGC
CGTTGTCGTT ACTCGTTGT
AATTGTTAAT GTTAATTGT
CGTTGTTAAT GTTCGTTGT
CATCATCAAA ACCCATCAT
AATCACGGCA GCCAATCAC
```
That is, interleaved. What a headache...

**Comment by josephwb** [comment]
Have this in hand, thanks to e0ece009bbd12ff6cf2420129bebd1c1eba63507.

**Comment by josephwb** [comment]
Things addressed with 330902aeb57ccd49c4ae721d21faf9c20f41ec29. Will close this after the merge.
